### PR TITLE
feat: openid connect spid and cie sdks

### PR DESCRIPTION
**Pull request by peppelinux**
## Description

This PR tackles:

* OIDC SDK for SPID and CIE in eng and it

## Checklist

* [x] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [x] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [x] Ready for review! :rocket:

### CODE REVIEW

File: _platforms/en/spid.md [hunk 1]
```diff
@@ -117,6 +117,27 @@ resources:
       url: https://github.com/italia/spid-ansible-shibboleth-example
       icon: github
       title: Example of configuration of Shibboleth with Nginx via an Ansible playbook
+  - OpenID Connect SDK:
```
Comment (ruphy) [inline _platforms/en/spid.md:120]:
```suggestion
  - OpenID Connect SDKs:
```

File: _platforms/en/spid.md [hunk 2]
```diff
@@ -24,19 +24,19 @@ owners:
...
       url: https://registry.spid.gov.it/identity-providers
       icon: github
-      desc: SAML Metadata of the SPID Identity Providers. Service Providers are required to keep metadata up-to-date
+      desc: SAML2 Metadata of the SPID Identity Providers. Service Providers are required to keep metadata up-to-date
     - title: Forms, regulation and documents
       url: https://www.agid.gov.it/it/piattaforme/spid
       icon: file
       desc: The AGID website contains forms, pricing, regulation and other documents about SPID
-  - SDK:
+  - SAML2 SDK:
```
Comment (ruphy) [inline _platforms/en/spid.md:39]:
```suggestion
  - SAML2 SDKs:
```

File: _platforms/en/spid.md [hunk 3]
```diff
@@ -117,6 +117,27 @@ resources:
       url: https://github.com/italia/spid-ansible-shibboleth-example
       icon: github
       title: Example of configuration of Shibboleth with Nginx via an Ansible playbook
+  - OpenID Connect SDK:
+    - title: SDK for Python
```
Comment (ruphy) [inline _platforms/en/spid.md:121]:
```suggestion
    - title: SDK for Python (Django)
```

File: _platforms/it/spid.md [hunk 4]
```diff
@@ -24,19 +24,19 @@ owners:
...
       url: https://registry.spid.gov.it/identity-providers
       icon: file
-      desc: Metadata degli Identity Provider, che i Service Provider devono mantenere aggiornati
+      desc: SAML2 Metadata degli Identity Provider, che i Service Provider devono mantenere aggiornati
     - title: Modulistica, circolari e deliberazioni
       url: https://www.agid.gov.it/it/piattaforme/spid
       icon: file
       desc: Sito di AGID contenente la modulistica, i listini, le convenzioni e le deliberazioni che riguardano SPID
-  - SDK:
+  - SAML2 SDK:
```
Comment (ruphy) [inline _platforms/it/spid.md:39]:
```suggestion
  - SDK SAML2:
```

File: _platforms/it/spid.md [hunk 5]
```diff
@@ -117,6 +117,27 @@ resources:
       url: https://github.com/italia/spid-ansible-shibboleth-example
       icon: github
       title: Esempio di configurazione di Shibboleth con Nginx tramite un playbook Ansible
+  - OpenID Connect SDK:
```
Comment (ruphy) [inline _platforms/it/spid.md:120]:
```suggestion
  - SDK OpenID Connect:
```

File: _platforms/it/spid.md [hunk 6]
```diff
@@ -117,6 +117,27 @@ resources:
...
+    - title: Python SDK
```
Comment (ruphy) [inline _platforms/it/spid.md:121]:
```suggestion
    - title: SDK per Python (Django) (beta)
```

File: _platforms/it/spid.md [hunk 7]
```diff
@@ -117,6 +117,27 @@ resources:
...
+      icon: github
+      url: https://github.com/italia/spid-cie-oidc-django
+      desc: SPID/CIE OIDC Federation è una collezione di applicazioni Django disegnate per rendere facile costruire e testare una Federazione Openid Connect.
+    - title: Nodejs SDK
```
Comment (ruphy) [inline _platforms/it/spid.md:125]:
```suggestion
    - title: SDK per Node.js (beta)
```

File: _platforms/it/spid.md [hunk 8]
```diff
@@ -117,6 +117,27 @@ resources:
...
+  - OpenID Connect SDK:
+    - title: Python SDK
+      icon: github
+      url: https://github.com/italia/spid-cie-oidc-django
+      desc: SPID/CIE OIDC Federation è una collezione di applicazioni Django disegnate per rendere facile costruire e testare una Federazione Openid Connect.
+    - title: Nodejs SDK 
+      icon: github
+      url: https://github.com/italia/spid-cie-oidc-nodejs
+      desc: SPID/CIE OIDC Federation Relying Party per Node.js
+    - title: AspNetCore SDK
```
Comment (ruphy) [inline _platforms/it/spid.md:129]:
```suggestion
    - title: SDK per AspNetCore (beta)
```

File: _platforms/it/spid.md [hunk 9]
```diff
@@ -117,6 +117,27 @@ resources:
...
+      icon: github
+      url: https://github.com/italia/spid-cie-oidc-aspnetcore
+      desc: SPID/CIE OIDC Federation SDK per AspNetCore
+    - title: PHP SDK
```
Comment (ruphy) [inline _platforms/it/spid.md:133]:
```suggestion
    - title: SDK per PHP (beta)
```

File: _platforms/it/spid.md [hunk 10]
```diff
@@ -117,6 +117,27 @@ resources:
...
+      icon: github
+      url: https://github.com/italia/spid-cie-oidc-php
+      desc: SPID/CIE OIDC Federation Relying Party per PHP
+    - title: Spring SDK
```
Comment (ruphy) [inline _platforms/it/spid.md:137]:
```suggestion
    - title: SDK per Java Spring (beta)
```

**Review by ruphy — APPROVED** [review]

**Comment by ruphy** [comment]
@peppelinux per me ci siamo, quando poi siamo pronti togli le label beta dagli SDK

**Comment by peppelinux** [comment]
@ruphy I just remvoed the beta label from the SDKs, I think we're ready to merge

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> fix: OpenID SDKs typos
- subject must not be sentence-case, start-case, pascal-case, upper-case</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9053b3ff9c9d897b5c8952648390cf7f2ab14e08
</p>

**Pull request merged**